yml/hsc.yaml

```diff
@@ -684,6 +690,18 @@ tables:
     description: Set to 1 for any fatal failure. Measured on g-band.
     mysql:datatype: BOOLEAN
     fits:tunit:
+  - name: g_sizeExtendedness
+    "@id": "#Object.g_sizeExtendedness"
+    datatype: double
+    description: Measure of likelihood to be a galaxy. Measured on g-band.
+    mysql:datatype: DOUBLE
+    fits:tunit:
+  - name: g_sizeExtendedness_flag
+    "@id": "#Object.g_sizeExtendedness_flag"
+    datatype: boolean
+    description: Set to 1 for any fatal failure. Measured on g-band.
```

Comment:
This should say what it is a fatal failure of.

Reply:
Hmm, many other plugins have similar description without going to the details.